ci/pipelines/deploy-to-production.yml

```diff
@@ -23,6 +23,64 @@ definitions:
     aws_ecr_registry_id: "((pay_aws_prod_account_id))"
     aws_region: eu-west-1
 
+  # Separate tasks for each combination of scenario/environment
+  smoke-test-create-card-payment-sandbox-on-production: &smoke-test-create-card-payment-sandbox-on-production
```

Comment:
I'm guilty of anchoring the original pr pipeline to within (or more likely beyond) an inch of its life and in hindsight it wasn't a fabulous idea (easier to write and modify but less easy to read 6 months later). I'm just wondering if these many anchors could be replaced with just one `&run-every-smoke-test` anchor since we seem to reference them all every time (unless I've miss-read it)?

Reply:
I tried doing that and the Concourse validation complained at me for having multiple tasks in the same anchor - very happy to squish them down if there's a way to do that?

Reply:
Maybe concourse validation is trying to protect us from ourselves. Was it valid yaml but the concourse `validate-pipeline` didn't like it?

Reply:
Yep. If I did something like:

```
smoke-test-run-all-on-production: &smoke-test-run-all-on-production
    - task: run_create_card_payment_sandbox-production
      file: pay-ci/ci/tasks/run-smoke-test.yml
      params:
        <<: *aws_assumed_role_creds
        AWS_REGION: "eu-west-1"
        SMOKE_TEST_NAME: "card_sandbox_prod"
    - task: run_create_card_payment_smartpay-production
      file: pay-ci/ci/tasks/run-smoke-test.yml
      params:
        <<: *aws_assumed_role_creds
        AWS_REGION: "eu-west-1"
        SMOKE_TEST_NAME: "card_smartpay_prod"
     [etc]
```

Then I'd get the following when trying to commit:

```
error: yaml: map merge requires map or sequence of maps as the value
```

Reply:
you could potentially solve this problem and speed everything up by running them `in_parallel`  all https://concourse-ci.org/jobs.html#in-parallel-step. That would allow you to have one anchor for the top level in parallel bit, which would contain all these bad boys

Reply:
Ahhh that's perfect. Thanks @mrlumbu!